packages/builder/yarn.lock

```diff
@@ -821,10 +821,17 @@
     lodash "^4.17.19"
     to-fast-properties "^2.0.0"
 
+<<<<<<< HEAD
```

Comment:
Sneaky merge conflict here!

Reply:
Oh wow! Thought I had sorted that one out 😬